test/performancetest/resources/config.json

```diff
@@ -4,16 +4,8 @@
 		"run_as_user": "root"
 	},
 	"metrics": {
-		"aggregation_dimensions": [
-			[
-				"InstanceId"
```

Comment:
Overall, aggregation with only InstanceID will return `InstanceID, AutoScalingGroupName, InstanceType, ImageId` and dimension of `InstanceID` which introduces extra dimension here. I'm fine with keeping it here too since I have updating it and decide to not using this feature for more visibility (it also does not make sense to combine multiple process together if using `aggregation_dimensions`)